src/pages/Younguk/List/List.scss

```
@@ -0,0 +1,196 @@
+* {
+  box-sizing: border-box;
+
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  display: flex;
+  list-style: none;
+} /* ul 리스트의 하위 태그들을 병렬로 표시해주기 위해 flex 속성을 적용하였다. */
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 50px;
+  margin: 3% 5% 3% 5%;
+  background-color: #f4f4f2;
+}/*상단 배너에 flex의 속성을 이용하여 내부 버튼들을 알맞게 레이아웃해줌. */
+
+.menu-banner {
+  display: flex;
+  align-items: center;
+  height: 50px;
+  margin: 2% 5% 2% 5%;
+  background-color: #f4f4f2;
+
+  .cold {
+    margin-left: 2%;
+    font-weight: bold;
+  }
+} /* nav와 양 옆 마진을 동일하게 해주고, 알맞게 디자인해줌 */
+
+.menu-list {
+  margin: 3% 5% 3% 5%;
+  
+  a {
+    font-weight: bold;
+  }
+
+  .linetemplate{
+    ul {
+      display: flex;
+      padding: 0;
+      flex-wrap: wrap;
+    } /* flex wrap을 통해 ul이 가로폭을 넘어사면 아래르 이동되게 해줌 */
+    li {
+      width: 25%;
+      height: 25%;
+      margin: 0;
+      margin-top:10px;
+      padding: 0;
+      text-align: center;
+      overflow: hidden;
+      
+    } /* ul 내의 li 태그들 width를 25%로 지정해주어 한 줄에 네개의 이미지와 리스트 병렬로 나열 */
+    
+    img {
+      width: 300px;
+      height: 300px;
+      padding: 0;
+      margin: 0;
+      transition: all 0.3s linear;
+      overflow: hidden;
+    } /* 후반부의 반응형 제작을 위해 이미지 크기를 px로 지정해주고 애니메이션 관련 속성을 지정해줌*/
+    
+    img:hover {
+      transform: scale(1.12);
+    } /* hover 시에 이미지가 1.12배 커지도록 설정해줌 */
+    
+  }
+} /* 사진과 텍스트가 포함된 모든 메뉴를 감싸주는 div 태그 */
+
+.logo {
+  font-family: "Dancing Script", cursive;
+  font-size: 35px;
+  text-decoration: none;
+  color: black;
+  margin-left: 3%;
+  font-weight: bold;
+} /* Webucks 로고를 디자인해줌 */
+
+li .topbanner-list {
+  padding: 10px;
+  color: black;
+  text-decoration: none;
+  color: gray;
+} /* 상단 배너 내의 버튼들 디자인 */
+
+
+.menu-banner img {
+  margin-left: 2%;
+  width: 30px;
+}
+
+.menu-banner .dcf {
+  margin-left: 2%;
+  font-size: 12px;
+  font-weight: bold;
+}
```

이 부분도 네스팅을 이용할 수 있지 않을까요?
```suggestion
.menu-banner {
    img {
       margin-left: 2%;
       width: 30px;
    }
    
    .dcf {
       margin-left: 2%;
       font-size: 12px;
       font-weight: bold;
    }
}
```